R/EstimationDataFromComparativeEffectivenessModel.R

```diff
@@ -0,0 +1,114 @@
+# @file EstimationDataFromComparativeEffectivenessModel.R
```

Comment:
I'm not following the logic of R files and their contents. I recommend merging this file with DataPull.R, and calling it DataPulls.R or some other simple name.

Reply:
Makes sense. Thank you